crates/typst-cli/src/world.rs

```diff
@@ -187,11 +195,14 @@ impl World for SystemWorld {
     }
 
     fn today(&self, offset: Option<i64>) -> Option<Datetime> {
-        let now = self.now.get_or_init(chrono::Local::now);
+        let today = match &self.today {
+            Today::Fixed(time) => time,
+            Today::System(time) => time.get_or_init(Utc::now),
```

Comment:
I think this is a change in behaviour? Previously `today(None)` returned the date in the current timezone, now it returns the date in UTC. Whichever is the intended behaviour, a doc comment on the method explaining it would be useful.

Reply:
I did not change the behavior, but I now added a comment explaining that: https://github.com/typst/typst/pull/3809/files#diff-bcbdea3af30438a4df7f95744e2520cc86473d36c58cf806e320d480cb49cad9R203-R207.